src/atoms/chips/filterChip.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ReactChild } from 'react';
+import { css } from 'styled-components';
+import { Styled } from '../../theme';
+import { IconWrapper, Text } from '../../index';
+
+export interface FilterChipProps {
+  children: ReactChild;
+  icon: JSX.Element;
+  isOn: boolean;
+  onClick: (e:any) => void;
+  id: number;
+}
+const Wrapper = Styled.div<{ isOn: boolean }>`
+  max-width: fit-content;
+  min-width: 79px;
+  box-sizing: border-box;
+  background: ${(props) => props.theme.containerAndCardShades.SHADE_PLUS_2};
+  padding: 4px 12px;
+  border-radius: 12px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  ${({ isOn }) => isOn && css`
```

Comment:
Why not move this up where you define background color already?

So you would have 

`background: ${({ isOn, theme }) => isOn ? theme.containerAndCardShades.NEUTRAL_SHADE_0 : theme.containerAndCardShades.SHADE_PLUS_2}`

am i missing something here?